tutor/config.py

```diff
@@ -358,3 +358,23 @@ def _check_preview_lms_host(config: Config) -> None:
             f'Warning: PREVIEW_LMS_HOST="{preview_lms_host}" is not a subdomain of LMS_HOST="{lms_host}". '
             "This configuration is not typically recommended and may lead to unexpected behavior."
         )
+
+
+@hooks.Actions.CONFIG_LOADED.add()
+def _check_run_meilisearch(config: Config) -> None:
+    """
+    In case RUN_MEILISEARCH is set to false, check if
+    MEILISEARCH_URL has been set to a non-default value.
+    If not, print a warning to notify the user.
+    """
+
+    run_meilisearch = get_typed(config, "RUN_MEILISEARCH", bool, True)
+    if not run_meilisearch:
+        meilisearch_url = get_typed(config, "MEILISEARCH_URL", str, "")
+        meilisearch_url_default = get_defaults()["MEILISEARCH_URL"]
```

Comment:
I'd like to avoid making another call to `get_defaults` here, which is slow. Can we just hardcode "meilisearch"?

Reply:
I'm never a big fan of hardcoding the values of defaults, because it means that if the default ever changes, you need to fix it in more than one place. Is there another (better) way to avoid the `get_defaults()` call?

Reply:
> I'm never a big fan of hardcoding the values of defaults, because it means that if the default ever changes, you need to fix it in more than one place.

[thoughts] While I agree with this, I try to weigh the effort involved in determining how many "changes" are needed. If it is needed in very few places, it can be okay to hardcode.

Reply:
I do agree with you Florian that hard-coding values is typically a bad practice. There are other ways to bypass this, but I wouldn't say they are "better". They include:

1) Your approach: calling `get_defaults()`, which is slow.
2) Implementing a `CONFIG_DEFAULTS_LOADED` action that would save a singleton copy of the (unrendered) config defaults, which would be quite complex.

If we ever forget to modify the meilisearch URL default, the warning you added will be present in the console stderr. Also, to make sure we don't forget to update this value, you could add a unit test.

Reply:
@fghaas Hi, following up on this PR. Thanks

Reply:
@fghaas Hi, checking up on this. Thanks